### PR TITLE
blueoi_seo module

### PR DESCRIPTION
**Pull request by doostinharrell**
initial blueoi_seo module that adds rel next and prev to pages for SEO purposes

### CODE REVIEW

File: modules/blueoi_seo/blueoi_seo.module [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?php
+
```
Comment (andyg5000) [inline modules/blueoi_seo/blueoi_seo.module:2]: needs @file docblock

File: modules/blueoi_seo/blueoi_seo.module [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ */
+function blueoi_seo_pager_previous($variables) {
+  $text = $variables['text'];
+  $element = $variables['element'];
+  $interval = $variables['interval'];
+  $parameters = $variables['parameters'];
+  global $pager_page_array;
+  $output = '';
+
+  // If we are anywhere but the first page
```
Comment (andyg5000) [inline modules/blueoi_seo/blueoi_seo.module]: needs punctuation

File: modules/blueoi_seo/blueoi_seo.module [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    else {
+      $output = theme('pager_link', array('text' => $text, 'page_new' => $page_new, 'element' => $element, 'parameters' => $parameters, 'attributes' => $attributes));
+    }
+  }
+
+  return $output;
+}
+
+/**
+ * Implements theme_pager_previous().
```
Comment (andyg5000) [inline modules/blueoi_seo/blueoi_seo.module]: incorrect docblock title

**Comment by doostinharrell** [comment]
Deeeeeerrrrrpp

## 

Dustin Harrell
Sent with Airmail

On November 16, 2016 at 2:34:59 PM, andyg5000 (notifications@github.com)
wrote:

## _@andyg5000_ commented on this pull request.

In modules/blueoi_seo/blueoi_seo.module
https://github.com/andyg5000/blueoi/pull/4#pullrequestreview-8891329:

> +
> -    // If the previous page is the first page, mark the link as such.
> -    if ($page_new[$element] == 0) {
> -      $output = theme('pager_first', array('text' => $text, 'element'
>   => $element, 'parameters' => $parameters, 'attributes' =>
>   $attributes));
> -    }
> -    // The previous page is not the first page.
> -    else {
> -      $output = theme('pager_link', array('text' => $text, 'page_new'
>   => $page_new, 'element' => $element, 'parameters' => $parameters,
>   'attributes' => $attributes));
> -    }
> -  }
>   +
> -  return $output;
>   +}
>   +
>   +/**
> - \* Implements theme_pager_previous().

incorrect docblock title

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/andyg5000/blueoi/pull/4#pullrequestreview-8891329, or mute
the thread

**Pull request merged**